en/deploy-on-aws-eks.md

```diff
@@ -262,25 +258,37 @@ The upgrade process does not finish immediately. You can watch the upgrade progr
 
 Before scaling out the cluster, you need to scale out the corresponding node group so that the new instances have enough resources for operation.
 
-The following example shows how to scale out the `tikv` group of the `<clusterName>` cluster to 4 nodes:
+This section describes how to scale out the EKS node group and TiDB components.
+
+### Scale out EKS node group
+
+The following example shows how to scale out the `tikv` group of the `${clusterName}` cluster to 4 nodes:
 
 {{< copyable "shell-regular" >}}
 
 ```shell
-eksctl scale nodegroup --cluster <clusterName> --name tikv --nodes 4 --nodes-min 4 --nodes-max 4
+eksctl scale nodegroup --cluster ${clusterName} --name tikv --nodes 4 --nodes-min 4 --nodes-max 4
 ```
 
-After that, execute `kubectl edit tc basic -n tidb-cluster`, and modify each component's `replicas` to the desired number of replicas. The scaling-out process is then completed.
-
 For more information on managing node groups, refer to [`eksctl` documentation](https://eksctl.io/usage/managing-nodegroups/).
 
+### Scale out TiDB components
+
+After scaling out the EKS node group, execute `kubectl edit tc basic -n tidb-cluster`, and modify each component's `replicas` to the desired number of replicas. The scaling-out process is then completed.
+
 ## Deploy TiFlash/TiCDC
 
+[TiFlash](https://docs.pingcap.com/tidb/stable/tiflash-overview) is the columnar storage extension of TiKV.
+
+[TiCDC](https://docs.pingcap.com/tidb/stable/ticdc-overview) is a tool for replicating the incremental data of TiDB by pulling TiKV change logs.
+
+The two components are *not required* in the deployment. This section shows a quick start example.
+
```

Comment:
We can change the Chinese version to the same 3-paragraph layout here

Reply:
Updated.